en/drafts/originals/understanding-creating-word-embeddings.md

```diff
@@ -24,135 +24,127 @@ doi: XX.XXXXX/phen0000
 
 {% include toc.html %}
 
-# Understanding and Creating Word Embeddings
+## Lesson Goals
+
+This lesson is designed to get you started with word embedding models. Through a primarily theoretical lense, you will learn how to prepare your corpus, read it into your Python session, and train a model. You will explore how word vectors work, how to interpret them, and how to perform some exploratory queries using them. We will provide some introductory code to get you started with word vectors, but the main focus will be on equipping you with fundamental knowledge and core concepts to use word embedding models for your own research. 
+
+While word embeddings can be implemented in many different ways using varying algorithms, this lesson does not aim to provide an in-depth comparison of word embedding algorithms (though we may at times make reference to them). It will instead focus on the `word2vec` algorithm, which has been used in a range of digital humanities and computational social science projects.[^1] 
```

Comment:
I have introduced endnotes (which you can find at the bottom of the file, at line 491). This relieves the lesson of too many in-text references to further research. I think it would be great to flesh out these endnotes with more information!